packages/components/src/core/styles/common/defaultTheme.ts

```diff
@@ -505,6 +637,9 @@ export interface Spaces {
 export type Spacings = Spaces;
 
 export interface Color {
+  900?: string;
+  800?: string;
+  700?: string;
```

Comment:
If we don't want all `Color` types to be able to climb this high we could make an `ExtendedColor` `interface` that extends `Color` and adds these three. Then it can be applied only where these values would be desired.